BetterPasswordPlugin.inc.php

```diff
@@ -304,4 +377,27 @@ function addUserSettings($hookname, $args) {
 		);
 		return false;
 	}
+        /**
+	 * Callback to fill cache with data, if empty.
+	 * @param $cache FileCache
```

Comment:
Are you enforcing that this is always a `FileCache`?  There are other implementations of `GenericCache` which might be used.